cura/Machines/MaterialManager.py

```diff
@@ -194,6 +198,11 @@ def initialize(self):
 
         self.materialsUpdated.emit()
 
+        favorites = self._application.getPreferences().getValue("cura/favorite_materials")
```

Comment:
I would move the favorites to a different place and `MaterialManager` only handles essential material lookup.

Reply:
We decided to keep this in. It's not so coupled with other things.